Jenkinsfile

```diff
@@ -11,48 +11,67 @@
 
 // TEST FLAG - to make it easier to turn on/off unit tests for speeding up access to later stuff.
 def runTests = true
+def failFast = false
 
 // Only keep the 10 most recent builds.
 properties([[$class: 'jenkins.model.BuildDiscarderProperty', strategy: [$class: 'LogRotator',
                                                                         numToKeepStr: '50',
                                                                         artifactNumToKeepStr: '20']]])
 
-node('java') {
-    timestamps {
-        // First stage is actually checking out the source. Since we're using Multibranch
-        // currently, we can use "checkout scm".
-        stage('Checkout') {
-            checkout scm
-        }
+def buildTypes = ['Linux', 'Windows']
```

Comment:
Please update this with a comment pointing to the `ci.adoc` to explain the importance of these specific labels